diffblue.yml

```diff
@@ -1,4 +1,2 @@
-branch: master
-language: java
-buildCmd: "mvn build"
-testCmd: "mvn cobertura:cobertura"
+buildCmd: "mvn test"
```

Comment:
Are there any datastax branches where it doesn't happen? If so, it's pretty easy to find the problem using `git bisect`. Give me a shout and I'll show you how.